### DISCUSSION TITLE
Schematic editor for myHDL

### DISCUSSION THREAD

**Discussion by eskiyerli** [myhdl/myhdl#418] [Ideas]
Somebody suggested that the Revolution EDA schematic editor can be used to import myHDL modules and create structural descriptions of systems. As Revolution EDA is also Python based that might be relatively straightforward. I want to understand if there is a need for such a graphical front-end for design entry among myHDL users. 
Any feedback is appreciated.

**Comment by josyb** [comment]
I visited your web-site briefly. Perhaps when I find more time I can take a deeper look.
Perhaps there may be interest in a graphical representation of a structural design to serve as documentation. But I doubt that MyHDL users would be interested in graphical  entry.

Best regards,
Josy

**Reply by eskiyerli** [reply]
Thanks for your response. What I have seen is myHDL is used for event driven system simulations. A GUI could make it similar to Simulink.

**Reply by olisnr** [reply]
i would like myHDL in Xschem

**Reply by eskiyerli** [reply]
You should then bring up this issue to Xschem developer.
